src/types/types.ts

```diff
@@ -0,0 +1,47 @@
+export interface IUser {
+  userId: number;
+  username: string;
+  accessToken: string;
+}
+
+export interface IUserLogin {
+  userId: number;
+  username: string;
+}
+
+export interface IUserRegistration {
+  username: string;
+  email: string;
+  password: string | number;
+  isAdmin: boolean;
```

Comment:
Bad name, I think i our case it should be instructor, and I'm not sure that we need all of those model right now, it should be extension of this module when we will be writing the part for that

Reply:
I changed isAdmin to isInstructor. I removed the modules that we don't need yet. Also, the names of the types will continue to change